### PR TITLE
Don't inject Bruteforce capability info in the webui

### PR DESCRIPTION
**Pull request by CarlSchwan**
This capability do DB access and as far I know is not used by the webui.
This removes one DB query for each page load.

### CODE REVIEW

**Comment by CarlSchwan** [comment]
if someone has a better name for the interface, it would be great...

**Comment by nickvergessen** [comment]
> if someone has a better name for the interface, it would be great...

Maybe reference "initial state" instead of "injecting".
If we would create all this just now I guess we would also turn it around and make the Initial state opt in? So that makes it a bit more complicated

**Comment by CarlSchwan** [comment]
> > if someone has a better name for the interface, it would be great...
> 
> Maybe reference "initial state" instead of "injecting". If we would create all this just now I guess we would also turn it around and make the Initial state opt in? So that makes it a bit more complicated

Renamed to IInitialStateExcludedCapability :)

**Review by skjnldsv — APPROVED** [review]

File: lib/public/Capabilities/IInitialStateExcludedCapability.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ */
+namespace OCP\Capabilities;
+
+/**
+ * Indicate that a capability should not be injected in the initial state
+ * of the page as it might be expensive to query and not useful for the
+ * webui.
+ */
+interface IInitialStateExcludedCapability {
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply lib/public/Capabilities/IInitialStateExcludedCapability.php:31]: The autoloaders are not up to date

[the remark this answers is not in the document]
Comment (CarlSchwan) [reply lib/public/Capabilities/IInitialStateExcludedCapability.php:33]: Should be good now

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**